### PR TITLE
Connect community event cards to database

### PR DESCRIPTION
**Pull request by susannahameyer**
Event cards within the community page now pull data from the db and calculate whether they are upcoming or past based on the current date.
I added the logic into the profile page because the components were edited, but we'll change the event getting logic to be profile-specific later and I'll consolidate the logic between the two so we're not repeating.
Next steps: bookmark star data is still hard-coded, and need to make community cover photo component pull from database. Also need to filter events so that we only show events where the current user's community is listed in the sponsors list.

### CODE REVIEW

File: volunta/components/CommunityProfileEventCard.js [hunk 1]
```diff
@@ -15,120 +17,125 @@ export default class CommunityProfileEventCard extends React.Component {
...
     this.state = {
-      title: props.title,
-      cover_url: props.cover_url,
-      org_name: props.org_name,
-      date: props.date,
       // whether or not a user has starred the event
       bookmarked: this._getBookmarked(),
-      // whether the event should appear in the upcoming list or not
-      comingUp: props.comingUp,
+      event: props.event,
```
Comment (lfvarela) [inline volunta/components/CommunityProfileEventCard.js]: Use 'events' directly from props like we discussed? You do this correctly in componentWillMount.

File: volunta/components/CommunityProfileEventCard.js [hunk 2]
```diff
@@ -15,120 +17,125 @@ export default class CommunityProfileEventCard extends React.Component {
...
+    });
+  }
+
   //TODO implement with db logic
   _getBookmarked = () => {
     return true;
   };
 
   render() {
+    const event = this.state.event;
```
Comment (lfvarela) [inline volunta/components/CommunityProfileEventCard.js]: Followup from first comment

File: volunta/screens/CommunityScreen.js [hunk 3]
```diff
@@ -1,66 +1,38 @@
...
   };
 
+  async componentDidMount() {
+    this._loadData();
+  }
+
+  _loadData = async () => {
+    const [upcomingEvents, pastEvents] = await getEventsForCommunity();
+    this.setState({
+      upcomingEvents: upcomingEvents,
```
Comment (lfvarela) [inline volunta/screens/CommunityScreen.js]: Just a style preference, but can change to setState({ upcomingEvents, pastEvents })

File: volunta/screens/ProfileScreen.js [hunk 4]
```diff
@@ -13,61 +13,37 @@ import InterestBubble from '../components/InterestBubble';
...
+
+  async componentDidMount() {
+    this._loadData();
+  }
+
+  //TODO: change this to be profile-specific and consolidate profile/community logic in a shared space
+  _loadData = async () => {
+    const [upcomingEvents, pastEvents] = await getEventsForCommunity();
+    this.setState({
+      upcomingEvents: upcomingEvents,
```
Comment (lfvarela) [inline volunta/screens/ProfileScreen.js]: same here

File: volunta/components/CommunityProfileEventCard.js (hunk 1 above)
Comment (susannahameyer) [reply volunta/components/CommunityProfileEventCard.js]: oh shit you right good catch

**Pull request merged**